addons/point_of_sale/models/account_move.py

```diff
@@ -17,6 +17,13 @@ def _stock_account_get_last_step_stock_moves(self):
             stock_moves += invoice.sudo().mapped('pos_order_ids.picking_id.move_lines').filtered(lambda x: x.state == 'done' and x.location_id.usage == 'customer')
         return stock_moves
 
+    @api.depends('pos_order_ids')
```

Comment:
By doing so, `_compute_amount` is called 3 times instead of 2 in a normal POS invoicing processing